### ISSUE TITLE
华为H122-373/H112-573华为5G-cpe-pro2（现存）固件汇总 - 风流云的博客

### ISSUE THREAD

**Issue by qyyd5g** [qyyd5g/qyyd5g.github.io#37]
https://qyyd5g.top/%E5%8D%8E%E4%B8%BAH122-373-573(%E7%8E%B0%E5%AD%98)%E5%9B%BA%E4%BB%B6%E6%B1%87%E6%80%BB/

**Comment by Jayr1279** [comment]
Hi I addedd you in wechat. I want to request H122-373 update firmware for C233. Thanks

**Comment by fagghghhg** [comment]
thanks

**Comment by fagghghhg** [comment]
Hi I addedd you in wechat. I want to request H122-373 update firmware for H612SP6C983. My Wechat ID is Jayr1279. Thanks

**Comment by q88sat** [comment]
مرحبا كيف يمكن ان اشتري تحديثات لراوتر هواوي 5G

**Comment by qyyd5g** [comment]
> @q88sat
> مرحبا كيف يمكن ان اشتري تحديثات لراوتر هواوي 5G
> 

wechat

**Comment by Fayed89** [comment]
I cant verify my account on wechat you provide an alternative ?

**Comment by estevao124** [comment]
Oi quero saber como posso te adicionar ao WeChat?

**Comment by qyyd5g** [comment]
> @Fayed89

no

**Comment by qyyd5g** [comment]
> @estevao124

my wechat  id: qyyd5g

**Comment by q88sat** [comment]
I could not register in wicht. The way is difficult or blocked in our countries. What is the solution? Do you have WhatsApp

**Comment by qyyd5g** [comment]
> @q88sat

no, only wechat，if you have alipay or QQ ，thay is other chinese software. We cannot use software and networks that are not approved by the  motherland..

**Comment by estevao124** [comment]
Eu tenho WhatsApp

Em ter, 24 de nov de 2020 08:06, q88sat <notifications@github.com> escreveu:

**Comment by estevao124** [comment]
Da para usar WhatsApp?

Em ter, 24 de nov de 2020 12:39, Estevao Oliveira <
estevaooliveira178@gmail.com> escreveu:

**Comment by estevao124** [comment]
É que eu preciso muito do firmware do roteador zte mf25a

Em qua, 25 de nov de 2020 05:52, Estevao Oliveira <
estevaooliveira178@gmail.com> escreveu:

**Comment by qyyd5g** [comment]
Is illegal in China

**Comment by estevao124** [comment]
Que pena o meu roteador mf25a está com EPROM corrompida e eu só consigo
consertar com este firmware

Em qua, 25 de nov de 2020 08:47, qyyd5g <notifications@github.com> escreveu:

**Comment by estevao124** [comment]
Da para me enviar o firmware do roteador mf25a por e-mail?

Em qua, 25 de nov de 2020 08:53, Estevao Oliveira <
estevaooliveira178@gmail.com> escreveu:

**Comment by kuw85** [comment]
could you add 
10.0.5.17(H612SP3C1367)
10.0.5.29(H612SP3C1367)
1367

**Comment by kuw85** [comment]
could you add
10.0.5.17(H612SP3C1367)
10.0.5.29(H612SP3C1367)
1367
can you recommended a website to unlock h122-373 ?

**Comment by fagghghhg** [comment]
قوه ماعرفت تشتري من صينين

‫أُرسلت من الـ iPhone‬

‫في 13‏/01‏/2021 الساعة 1:01 م، كتب/كتبت ‏kuw85 ‏<notifications@github.com>:‬

﻿

could you add
10.0.5.17(H612SP3C1367)
10.0.5.29(H612SP3C1367)
1367
can you recommended a website to unlock h122-373 ?

**Comment by qyyd5g** [comment]
> @kuw85

I don't know what you mean by these versions, I don't have these two files. There is currently no recommended unlocking method.

**Comment by kuw85** [comment]
> > @kuw85
> 
> I don't know what you mean by these versions, I don't have these two files. There is currently no recommended unlocking method.

i mean can you add new updates for this router C1367

we wants new firmware like the c00 does

**Comment by qyyd5g** [comment]
> @kuw85

Huawei has not updated the version in your country yet, so you have to wait.

**Comment by fagghghhg** [comment]
kuwait

‫أُرسلت من الـ iPhone‬

‫في 13‏/01‏/2021 الساعة 7:36 م، كتب/كتبت ‏qyyd5g ‏<notifications@github.com>:‬

﻿

@kuw85<https://github.com/kuw85>

Huawei has not updated the version in your country yet, so you have to wait.

**Comment by fagghghhg** [comment]
> @kuw85
> 
> 
> could you add 
> 10.0.5.17(H612SP3C1367)
> 10.0.5.29(H612SP3C1367)
> 1367
> )
> sned me this 10.0.5.29(H612SP3C1367) 
thank you

**Comment by fagghghhg** [comment]
英语0H122-373-573_11.0.2.7(H329SP61C00)_UTPS_11.0.2.1(W2SP68C03)-sec.ZIP
you have please send me

‫أُرسلت من الـ iPhone‬

‫في 16‏/01‏/2021 الساعة 8:39 ص، كتب/كتبت ‏fahad nar ‏<F_777_F@hotmail.com>:‬

﻿
kuwait

‫أُرسلت من الـ iPhone‬

‫في 13‏/01‏/2021 الساعة 7:36 م، كتب/كتبت ‏qyyd5g ‏<notifications@github.com>:‬

﻿

@kuw85<https://github.com/kuw85>

Huawei has not updated the version in your country yet, so you have to wait.

**Comment by fagghghhg** [comment]
<https://www.icloud.com/attachment/?u=https%3A%2F%2Fcvws.icloud-content.com%2FB%2FAaSt20Q522RCHKolc0ZOxtULUA7qAdoiwtATs0yT5KUn79B0Ttw1CZU-%2F%24%7Bf%7D%3Fo%3DAgtBk_Hi258bZVgmQuMcr0jh0kgMS7XXzSanRaIY5j7Q%26v%3D1%26x%3D3%26a%3DCAog1g90VBejiNfvM9D-AHAMP9PedJx2skKmCj-Wv8GinQASdhCp8b2n8S4YqYG5-_ouIgEAKgkC6AMA_2RGIZhSBAtQDupaBDUJlT5qJZXY4Iu6UhRe5UxCzeu2CP0Pe7nMoTXcK2cHgJMd3g7hSF0GfnZyJe2ivvfw_jh6sYz47Uecgz1lITYRjGNZoGW6XWI6VxLhUKKRA2w%26e%3D1613555974%26fl%3D%26r%3D6FFF9888-BF01-4A23-BD7C-FB56BCCD180B-1%26k%3D%24%7Buk%7D%26ckc%3Dcom.apple.largeattachment%26ckz%3D3CDB3D75-4A2E-4502-8712-639241A23688%26p%3D68%26s%3D-4xloUs5tZDaebXLSQBDD4uwK-8&uk=H5cObwG9JbyCnuutNYMcIg&f=MIX-H122-373-573_UPDATE_10.0.5.1%28H612SP1C1367%29.BIN&sz=74580252>
تنزيل المرفق
متوفر حتى 17‏/02‏/2021

**Comment by Locutus1000** [comment]
您好如何為華為H122-373路由器獲取（下載）固件H122-373 10.0.5.29（H612SP1C00）

Nín hǎo rúhé wèi huáwèi H122-373 lùyóuqì huòqǔ (xiàzài) gùjiàn H122-373 10.0.5.29(H612SP1C00)

**Comment by Locutus1000** [comment]
Hello I addedd you in wechat. I want to request H122-373 update firmware for
- MIX-H122-373-573_UPDATE_10.0.5.13(H612SP5C1217).BIN.
- H122-373-573_10.0.5.29(H612SP3C00)_UTPS_10.0.5.1(W2SP32C03)-sec.ZIP
- MIX-H122-373-573_UPDATE_10.0.5.29(H612SP1C1367)_WEBUI_10.0.5.1(W2SP2C1367).BIN
My Wechat ID is LocMani
Thanks
Best Regards

**Comment by radikal357** [comment]
@qyyd5g
do you have h122-373-A firmware?
here is my wechat: wxid_19nseniflwg212

**Comment by brithospoti990** [comment]
I have "H122-373 11.0.5.1(H336SP3C00)_Firmware_general_05018MLQ.zip" file. How can I get separate Bin files for software and WBUI Bin? Please help

**Comment by p360h** [comment]
> I have "H122-373 11.0.5.1(H336SP3C00)_Firmware_general_05018MLQ.zip" file. How can I get separate Bin files for software and WBUI Bin? Please help
Hi, Could you please send me this framware "H122-373 11.0.5.1(H336SP3C00)_Firmware_general_05018MLQ.zip" ?
p360h@yahoo.com
Telegram : @hpeyman
